app/config/adherent.yml

```diff
@@ -26,3 +26,10 @@ parameters:
         mouvement: Je ne me reconnais plus dans les actions du mouvement En Marche !
         comite: Je suis en désaccord avec l’activité de mon comité local
         autre: Autre
+    user_unregistration_reasons:
```

Comment:
Let's use the translator instead of this ? 
We have something similar for the report feature with some reasons to check and it uses Translator. You can check it out. It's far easier to manage then.